### PR TITLE
feat(zimbra): technical base of zimbra project

### PR DESCRIPTION
**Pull request by ghyenne**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/zimbra` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-13039 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
I added the technical base of zimbra project

### CODE REVIEW

File: packages/manager/apps/zimbra/package.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  },
+   "dependencies": {
+    "@ovh-ux/manager-config": "^7.2.0",
+    "@ovh-ux/manager-core-api": "*",
+    "@ovh-ux/manager-react-core-application": "*",
+    "@ovh-ux/manager-core-utils": "*",
+    "@ovh-ux/shell": "^3.0.0",
+    "@tanstack/react-query": "^5.8.3",
+    "@tanstack/react-query-devtools": "^5.8.3",
+    "@ovhcloud/ods-common-theming": "^16.6.0",
```
Comment (aboungnaseng-ovhcloud) [inline packages/manager/apps/zimbra/package.json:31]: we need to update ods version 17.1.0

Comment (ghyenne) [reply packages/manager/apps/zimbra/package.json]: It's applied. I changed the package.json and the imports

**Comment by aboungnaseng-ovhcloud** [comment]
Hi, Can you use dashboard component inside react manager lib ?

File: packages/manager/apps/zimbra/src/components/layout-helpers/Listing/TableContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
+import React from 'react';
```
Comment (aboungnaseng-ovhcloud) [inline packages/manager/apps/zimbra/src/components/layout-helpers/Listing/TableContainer.tsx:1]: use datagrid of manager components

**Review by vovh — APPROVED** [review]

**Pull request closed without merging**